jobs/e2e_runner.groovy

```diff
@@ -8,7 +8,7 @@ import utilities.StatusUpdater
   isMaster = config.type == 'master'
   isPR = config.type == 'pr'
   name = isMaster ? "e2e-runner" : "e2e-runner-${config.type}"
-  dockerPush = isPR ? 'docker-immutable-push' : 'docker-mutable-push'
+  dockerPush = 'docker-push'
```

Comment:
I think we still want to _only_ push the immutable tag for a PR (so `canary` doesn't get a potentially flawed PR change), but we do want to push both immutable and mutable on a master commit (both the immutable tag for the actual merge commit and the updated `canary` tag)... something like:
```
dockerPush = isPR ? 'docker-immutable-push' : 'docker-push'
```
(This is what we do in our existing `component_job.groovy` https://github.com/deis/jenkins-jobs/blob/master/jobs/component_jobs.groovy#L98)